### PR TITLE
Works section and about me section

### PR DESCRIPTION
**Pull request by henokkhm**
This pull request adds the following features:

- Add works section and previous projects' cards and screenshots
- Refactor shared styles: primary button and social media links
- Add primary button 
- Add an about-me section with programming languages, frameworks, and skills sub-sections

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -84,23 +85,264 @@ <h2 class="hero__social-media-links__title">Let&apos;s connect</h2>
...
+        <!-- About me summary -->
+        <ul class="about-me__summary">
+          <li>
+            <div class="about-me__summary__header">
+              <span>Languages</span>
+              <img src="assets/icons/caret.svg" alt="expand-button" />
+            </div>
+            <ul class="about-me__languages-list">
+              <li class="about-me__language-item">
+                <img src="assets/icons/js.png" alt="JavaScript icon" />
```
Comment (iambenkis) [inline src/index.html]: - [x] Kindly adjust the margin bottom bettween the `Langaues` and the first item 👍 , to stick to the Figma template

| Your design | Figma template |
| :---: | :---: |
| Capture | Capture 4 |

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @henokkhm 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (henokkhm) [reply src/index.html]: @iambenkis Thank you for your feedback

Comment (stanleySimeon) [inline src/index.html]: According to the Figma template, kindly remove the gray line at the top of the first `about-me__language-item`.

Screenshot 2023-04-28 at 07 10 38

File: src/index.html [hunk 2]
```diff
@@ -84,23 +85,264 @@ <h2 class="hero__social-media-links__title">Let&apos;s connect</h2>
...
+        <ul class="works__cards-list">
+          <!-- Project-1 Card -->
+          <li class="works__project-card">
+            <!-- Project Screenshot -->
+            <img
+              class="works__project-card__screenshot"
+              src="assets/project-screenshots/proj-1.png"
+              alt="Screenshot of Tonic project"
+            />
+
```
Comment (stanleySimeon) [inline src/index.html:110]: I suggest removing the white spaces in your code, by following this your project will look more. professional. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @henokkhm 
 
Good job so far on implementing the change required by the previous reviewer. There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of GitHub flow ✔️
- Linter checks are passed ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 3]
```diff
@@ -93,39 +156,146 @@ body {
...
+  padding: 2em 0;
+  margin: 0;
+}
 
-/* Social Media Links Styles */
+.about-me__summary__header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.75em 1.5em;
+  color: var(--black);
```
Comment (Yerimah) [inline src/style.css]: - [x] Hey @henokkhm 👋 , you have done very well implementing the project requirements but there is still some adjustment to make your project look like the Figma template design. I will suggest you adjust the margin left for the `class="about-me__summary__header expanded" ` as defined in the original design.

> Figma Template
Screenshot 2023-04-28 at 18 32 10

> Your design
Screenshot 2023-04-28 at 18 30 46

File: src/index.html [hunk 4]
```diff
@@ -40,19 +40,20 @@
       <!-- Hero -->
```
Comment (Yerimah) [inline src/index.html:42]: - [ OPTIONAL] Always wrap your logo in an `<a>` tag to make it clickable for semantic and conventional reasons, and also to improve the user experience.

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES ♻️ 

Hi @henokkhm, 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✅ 
- Good commit messages ✅ 
- Descriptive PR title and summary ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 3 above)
Comment (henokkhm) [reply src/style.css]: @Yerimah @iambenkis  @stanleySimeon  Is there any tool that I can use to automatically compare the Figma design with my implementation, or do I have to look for these discrepancies manually?

**Review by pinkmoon25 — APPROVED** [review]
Hi @henokkhm , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter Checks :heavy_check_mark: 
Github flow :heavy_check_mark: 
Changes Implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**